packages/usa-range/src/test/sr-callout--preposition.spec.js

```diff
@@ -0,0 +1,74 @@
+const fs = require("fs");
+const path = require("path");
+const assert = require("assert");
+const range = require("../index");
+
+const TEMPLATE = fs.readFileSync(
+  path.join(__dirname, "./sr-callout--preposition.spec.html")
+);
+
+const EVENTS = {};
+
+/**
+ * send an change event
+ * @param {HTMLElement} el the element to sent the event to
+ */
+EVENTS.change = (el) => {
+  el.dispatchEvent(new KeyboardEvent("change", { bubbles: true }));
+};
+
+const rangeSliderSelector = () => document.querySelector(".usa-range");
+
+const tests = [
+  { name: "document.body", selector: () => document.body },
+  { name: "range slider", selector: rangeSliderSelector },
+];
+
+tests.forEach(({ name, selector: containerSelector }) => {
+  describe(`Range slider component initialized at ${name}`, () => {
```

Comment:
All range unit tests have the same description. It can be hard to parse which test is for which. Can you make these more descriptive? It might be helpful to follow the model in the unit tests for date picker.

Reply:
Added some clarity to unit test names in [39cca2b](https://github.com/uswds/uswds/pull/5472/commits/39cca2ba6addbabb78c58cc4728800d1beea3fbe). Thanks for the reference!